ruleset/sca/darwin/18/cis_apple_macOS_10.14.yml

```diff
@@ -0,0 +1,885 @@
+    rationale: "Patches need to be applied in a timely manner to reduce the risk of vulnerabilities being exploited"
+    remediation: "Open a terminal session and enter the following command to enable install system data files and security updates: sudo defaults write /Library/Preferences/com.apple.commerce AutoUpdate - bool TRUE"
+    compliance:
+      - cis: ["1.6"]
+    condition: all
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.commerce AutoUpdate -> 1'
+
+# jal 
+# 2.1.1 Turn off Bluetooth, if no paired devices exist (Automated) 
+  - id: 17006
+    title: "Turn off Bluetooth, if no paired devices exist"
+    description: "Bluetooth devices use a wireless communications system that replaces the cables used by other peripherals to connect to a system. It is by design a peer-to-peer network technology and typically lacks centralized administration and security enforcement infrastructure."
+    rationale: "Bluetooth is particularly susceptible to a diverse set of security vulnerabilities involving identity detection, location tracking, denial of service, unintended control and access of data and voice channels, and unauthorized device control and data access."
+    remediation: "Open a terminal session and enter the following command to disable bluetooth: sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0 && sudo killall -HUP blued"
+    compliance:
+      - cis: ["2.1.1"]
+    condition: any
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.Bluetooth ControllerPowerState -> 0'
+      - 'c:system_profiler SPBluetoothDataType -> r:Connectable: Yes'
+
+# jal
+# 2.1.2 Show Bluetooth status in menu bar (Automated) 
+  - id: 17007
+    title: "Show Bluetooth status in menu bar"
+    description: "By showing the Bluetooth status in the menu bar, a small Bluetooth icon is placed in the menu bar. This icon quickly shows the status of Bluetooth, and can allow the user to quickly turn Bluetooth on or off."
+    rationale: "Enabling \"Show Bluetooth status in menu bar\" is a security awareness method that helps understand the current state of Bluetooth, including whether it is enabled, Discoverable, what paired devices exist and are currently active."
+    remediation: "Open a terminal session and enter the following command to enable Bluetooth status in the menu bar. sudo -u <username> defaults write /Users/<username>/Library/Preferences/com.apple.systemuiserver menuExtras - array-add \"/System/Library/CoreServices/Menu Extras/Bluetooth.menu\""
+    compliance:
+      - cis: ["2.1.2"]
+    condition: any
+    rules:
+      - 'c:defaults read /Users/<username>/Library/Preferences/com.apple.systemuiserver menuExtras -> r:Bluetooth.menu'
```

Comment:
Don't you get this output due to the inexistence of `<username>` as a valid user?
```
# defaults read /Users/<username>/Library/Preferences/com.apple.systemuiserver menuExtras
sh: username: No such file or directory
```

SCA is not prepared to replace `<username>` by the needed user when running the command